killbill-integration-tests/core/test_account.rb

```diff
@@ -132,54 +132,5 @@ def test_account_update
 
     end
 
-    def test_emails_notifications
```

Comment:
⚠️ It looks like the conflict resolution led to this being deleted.

Might be easier to cherry-pick the good commit (https://git-scm.com/docs/git-cherry-pick) or rebase it (https://git-scm.com/book/en/v2/Git-Branching-Rebasing) on top of a clean branch.